### ISSUE TITLE
Gitlab project with pipeline now private

### ISSUE THREAD

**Issue by palfrey** [jdrouet/jolimail#656]
The README links to [a gitlab pipeline](https://gitlab.com/jeremie.drouet/jolimail/-/commits/main) which now appears to be private?

**Comment by jdrouet** [comment]
yeah, the link between GitHub and gitlab broke. I will rewrite the CI configuration to use GitHub actions
